bedrock/firefox/templates/firefox/whatsnew/base.html

```diff
@@ -2,23 +2,9 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 
-{% extends "firefox/base/base-resp.html" %}
+{% extends "firefox/base/base-protocol.html" %}
 
 {# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
 {% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
 {% block gtm_page_id %}data-gtm-page-id="/firefox/whatsnew/"{% endblock %}
-
-{% block extrahead %}
-  {% if switch('firefox-update-notification-modal', ['en-US']) %}
-    {{ css_bundle('firefox-update-notification-firstrun-whatsnew') }}
```

Comment:
Note for reviewer: I'm not carrying this old notification over, since it's not being shown currently ans has never been ported to Protocol.

Reply:
👍